libtransmission/bandwidth.cc

```diff
@@ -223,7 +225,7 @@ void tr_bandwidth::allocate(uint64_t period_msec)
     // allocateBandwidth () is a helper function with two purposes:
     // 1. allocate bandwidth to b and its subtree
     // 2. accumulate an array of all the peerIos from b and its subtree.
-    this->allocate_bandwidth(TR_PRI_LOW, period_msec, refs);
+    this->allocate_bandwidth(TR_PRI_NONE, period_msec, refs);
```

Comment:
I'm thinking maybe adding a new value in the public API just to use it as a placeholder is not a good practice. Maybe something like this will be better, and then it will make sense to backport. WDYT?

```suggestion
    allocate_bandwidth(std::numeric_limits<tr_priority_t>::max(), period_msec, refs);
```

Reply:
It probably is better practice to change `transmission.h` as little as possible in the 4.0.x series